api/MediaSession.json

```diff
@@ -270,7 +274,8 @@
               }
             ],
             "firefox_android": {
-              "version_added": false
+              "version_added": "82",
```

Comment:
@ddbeck With respect to your comment on `MediaPositionState` here: https://github.com/mdn/browser-compat-data/pull/6845#discussion_r521497837

I have deleted the specific doc for the `MediaPositionState`. There is no code change here, so arguably we could just ignore adding anything for this - I added it because I though it was an omission. If we do need subfeature documentation, could you point me at a good example of how this would be done in BCD that I can copy.

Reply:
A quick search turned up something like `api.HTMLOrForeignElement.focus.preventScroll_option` ([MDN](https://developer.mozilla.org/en-US/docs/Web/API/HTMLOrForeignElement/focus)). The equivalent here would be something like `api.MediaSession.setPositionState.{duration,playbackRate,position}`, or, to preserve the top-level `MediaPositionState`, `api.MediaSession.setPositionState.MediaPositionState.{duration,playbackRate,position}`.

Also an unrelated heads up: you don't need to rebase PRs on BCD. We squash most merges anyway.

Reply:
Thanks @ddbeck . I think I've done this bit "as expected" now. Note again that I've copied in the same note as we discussed above - and I'm not sure this is appropriate within this level of nesting.